core/translations/en.json

```diff
@@ -81,7 +81,7 @@
     "bitcoin__unverified_external_inputs": "The transaction contains unverified external inputs.",
     "bitcoin__valid_signature": "The signature is valid.",
     "bitcoin__voting_rights": "Voting rights to:",
-    "brightness__title": "Set brightness",
+    "brightness__title": "Change display brightness",
```

Comment:
@obrusvit do you know whether we need to do anything when an english string changes?

Reply:
I'm not sure. In this case, it seems unnecessary. WDYT @Hannsek

Reply:
If it is not a big pain, please change it. 🙏🏻

Reply:
Will need new translations, for sure. But we are adding new strings anyway, so I think it's no big deal.